lib/hyperglot/hyperglot.yaml

```
@@ -11453,7 +11453,7 @@ zsm:
   name: Standard Malay
   orthographies:
   - autonym: Malaysia
-    auxiliary: \'
+    auxiliary: '
```

This is a syntax error: because a value starting with a quote character starts a string in YAML, it needs to be escaped. The backslash is not part of the value. If you are ending up with that value showing up somewhere it is because an invalid YAML parser is being used to read this file and a bug should be filled and fixed about whatever is being used to parse the YAML that is now following the YAML spec.

---

Ammend that, there is another issue here. The quote does need to be escaped, but in isolation I don't think that's the right way to do it. I think this is what needs to happen:

```suggestion
    auxiliary: "'"
```